src/layout/section/heroSection/busHeroSection.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { color, componentSizes, font, responsiveBreakpoints, space } from '../../../_utils/branding'
+import { TextDisplay1 } from '../../../typography/display1'
+import { TextTitle } from '../../../typography/title'
+
+export type BusHeroSectionProps = {
+  className?: string
+  heroImageUrl: string
+  heroText?: string
+  heroDescription?: string
+  bottomElement?: React.ReactElement
+}
+
+function BusHeroSection({
+  className,
+  heroText,
+  heroDescription,
+  bottomElement,
+}: BusHeroSectionProps) {
+  return (
+    <div className={className}>
+      <div className="hero-image" aria-hidden="true" />
+
+      <div className="hero-info">
+        <div className="hero-info-title-wrapper">
+          <h1 className="hero-info-title">
+            <TextDisplay1 className="hero-info-title--display1">{heroText}</TextDisplay1>
+          </h1>
+
+          {heroDescription && (
+            <p className="hero-info-text">
+              <TextTitle className="hero-info-text-title">{heroDescription}</TextTitle>
+            </p>
+          )}
+        </div>
+
+        <div className="hero-content">{bottomElement}</div>
+      </div>
+    </div>
+  )
+}
+
+const StyledBusHeroSection = styled(BusHeroSection)`
+  & {
+    background-color: ${color.white};
+    position: relative;
+    z-index: 1;
+    height: 80vh;
+    margin-bottom: ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  & .hero-image {
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 80%;
+    background-size: cover;
+    background-position: center;
+    background-image: ${props => `url(${props.heroImageUrl})`};
+
+    /* Hide the shadow. */
+    overflow: hidden;
+  }
+
+  & .hero-image::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 80%;
+    display: block;
+    background: linear-gradient(
+      0deg,
+      rgba(0, 0, 0, 0.4) 72%,
+      rgba(0, 0, 0, 0.2) 85%,
+      rgba(0, 0, 0, 0) 100%
+    );
+  }
+
+  & .hero-info {
+    width: 100%;
+    max-width: ${componentSizes.largeSectionWidth};
+    padding: 0 ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  & .hero-info-title-wrapper {
+    text-align: center;
+  }
+
+  & .hero-info-title {
+    margin: 0;
+    padding-top: ${space.m};
+    padding-bottom: ${space.m};
+  }
+
+  & .hero-info-title--display1 {
+    color: ${color.white};
+  }
+
+  & .hero-info-text {
+    margin: 0;
+  }
+
+  & .hero-info-text-title {
+    color: ${color.white};
+  }
+
+  & .hero-content {
+    width: 100%;
+    margin-top: ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
```

Comment:
nit for better readability:  the variations on style should be contained inside the component itself. It's easier to understand what changes on that component when moving breakpoints and easier for refactor/maintenance

eg:
```suggestion
 & .hero-image {
     position: absolute;
    z-index: -1;
    top: 0;
    left: 0;
    width: 100%;
    height: 80%;
    background-size: cover;
    background-position: center;
    background-image: ${props => `url(${props.heroImageUrl})`};
    /* Hide the shadow. */
    overflow: hidden;
    
 @media (${responsiveBreakpoints.isMediaLarge}) {
      height: 100%;
  }
    }
  
```

Reply:
I just renamed the file, but I agree it could be improved.
I'll refactor it for a better usage of styled-components in an other PR, if it's ok with you.